synfig-studio/src/gui/states/state_draw.cpp

```diff
@@ -918,15 +1008,33 @@ StateDraw_Context::event_stroke(const Smach::event& x)
 
 	get_work_area()->add_stroke(event.stroke_data,synfigapp::Main::get_outline_color());
 
+	bool is_mirror_data_valid = false;
+	if (event.mirrored_stroke_data && event.mirrored_stroke_data->size() > 1) {
+		if ((event.stroke_data->front()-event.mirrored_stroke_data->front()).mag_squared() < 1/40.) {
```

Comment:
I don't understand what `1/40.` means, so I suggest assigning that value to a named constant. Something like this:
```
constexpr float angle = 1/40.f;
```

Reply:
Whoa I put that arbitrary number and forgot to look at it: it is to check if to vertices are close enough to glue them - it is the maximum acceptable (squared) distance between them to merge them together.

I don't know how to "calculate" or "choose" this value :(

Reply:
Added a named constant and some comments there too.